shared/common-adapters/text.native.js

```diff
@@ -130,12 +136,12 @@ function _getStyle(
   type: TextType,
   backgroundMode?: Background = 'Normal',
   lineClampNum?: ?number,
-  clickable?: ?boolean
+  clickable?: ?boolean,
+  forceUnderline: boolean
 ) {
-  if (backgroundMode === 'Normal') return null
```

Comment:
@chrisnojima Is there a reason why we disabled underline on mobile? I removed this so the underline below would work.

Reply:
Related: I think with this change, on mobile we'll start underlining links, while previously we didn't automatically underline links on mobile.

Reply:
I ended up making it not automatically underline links; just when `underline={true}` is provided.